src/composables/sdk13.ts

```diff
@@ -203,6 +215,26 @@ export function useSdk13({ store }: IDefaultComposableOptions) {
     };
   }
 
+  if (watchNetwork) {
+    watch(
+      () => activeNetwork.value,
+      async (newValue, oldValue) => {
+        if (sdkBlocked || isEqual(newValue, oldValue)) {
+          return;
+        }
+        try {
+          sdkBlocked = true;
+          if (oldValue) {
+            sdk.pool.delete(oldValue.name);
+          }
+          sdk.addNode(newValue.name, (await createNodeInstance(newValue.url))!, true);
+        } finally {
+          sdkBlocked = false;
+        }
+      },
+    );
+  }
```

Comment:
I have moved the watchers to individual files, and dedicated watchers for background and popup. The approach with on-demand resetNode doesn't work as expected. 
When you are on the network page and change the network and navigate back to the dashboard, there  are multiple `getSdk` methods fired simultaneously, resulting in a `Duplicate node, node already exists error`. This could be addressed by a flag, due to the parallel execution of the components this is a poor approach to solving this problem, and expects the first `getSdk` call rest of the calls need to wait until the node has been updated. This makes the dashboard rendering really slow. Hence the source of truth remains in the watchers, I guess this also improves the readability too given we are gonna use too many nonpure variables in the `getSdk` method.